integration_tests/src/main/python/window_function_test.py

```diff
@@ -272,6 +272,39 @@ def do_it(spark):
                 .withColumn('row_num', f.row_number().over(baseWindowSpec))
     assert_gpu_and_cpu_are_equal_collect(do_it, conf={'spark.rapids.sql.hasNans': 'false'})
 
+
+lead_lag_array_data_gens = [ArrayGen(int_gen, max_length=10),
+                            ArrayGen(ArrayGen(int_gen, max_length=10), max_length=10),
```

Comment:
nit: Could we have a few more types than just ints?  So that we feel that we have covered more possible types.

Reply:
Done